pages/instances/troubleshooting/cant-connect-ssh.mdx

```diff
@@ -34,7 +34,7 @@ When you create your Instance, check step 6 of the creation wizard and make sure
 
 - You can view and manage your public SSH keys from the [credentials section of the console](https://console.scaleway.com/project/credentials).
 - Credentials are specific to each [Project](/organizations-and-projects/concepts/#project) of your Organization.
-- If you [add a new public key to your Project credentials](/organizations-and-projects/how-to/create-ssh-key/#how-to-upload-the-public-ssh-key-to-the-scaleway-interface) **after** creating a new Instance, it will not automatically be uploaded to the existing Instance. Scaleway only uploads the public SSH keys specified at the time of the Instance's creation. To add a new public SSH key after that point, you will need to connect to your Instance and add the additional public key yourself. Public keys are stored by default in `~/.ssh` with a filename similar to `id_rsa.pub`.
+- If you [add a new public key to your Project credentials](/organizations-and-projects/how-to/create-ssh-key/#how-to-upload-the-public-ssh-key-to-the-scaleway-interface) **after** creating a new Instance, it will not automatically be uploaded to the existing Instance. You can either reboot your Instance to transfer the new key to it during the boot process or manually run the command `scw-fetch-ssh-keys --upgrade` to download the new key to the Instance.
```

Comment:
```suggestion
- If you [add a new public key to your Project credentials](/organizations-and-projects/how-to/create-ssh-key/#how-to-upload-the-public-ssh-key-to-the-scaleway-interface) **after** creating a new Instance, it will not automatically be uploaded to the existing Instance. You can either reboot your Instance to transfer the new key to it during the boot process or manually run the command `scw-fetch-ssh-keys --upgrade` to upload the new key to the Instance.
```